atc/db/resource.go

```diff
@@ -384,6 +384,14 @@ func (r *resource) Versions(page Page) ([]atc.ResourceVersion, Pagination, bool,
 		WHERE r.id = $1 AND r.resource_config_scope_id = v.resource_config_scope_id AND v.check_order != 0
 	`
 
+	if len(versionFilter) != 0 {
+		filterJSON, err := json.Marshal(versionFilter)
+		if err != nil {
+			return nil, Pagination{}, false, err
+		}
+		query += fmt.Sprintf(` AND version @> '%s'`, string(filterJSON))
```

Comment:
We should use a param here to be safe. Someone could submit JSON with a `'` in it.

Reply:
Ah thx for pointing it out!  Updated to use sql query param.